src/Symfony/Component/Console/Command/CompleteCommand.php

```diff
@@ -31,16 +31,19 @@ final class CompleteCommand extends Command
     protected static $defaultName = '|_complete';
     protected static $defaultDescription = 'Internal command to provide shell completion suggestions';
 
-    private static $completionOutputs = [
-        'bash' => BashCompletionOutput::class,
-    ];
+    private $completionOutputs;
 
     private $isDebug = false;
 
+    public function __construct(array $completionOutputs = [])
+    {
+        $this->completionOutputs = $completionOutputs ?: ['bash' => BashCompletionOutput::class];
```

Comment:
should we add just a fallback, or *merge* the core ones with custom ones, so that when the core adds support for a new type, it is also available in projects registering a custom one ?